editor/components/post-publish-button/index.js

```diff
@@ -82,6 +82,7 @@ export default compose( [
 			visibility: getEditedPostVisibility(),
 			isSaveable: isEditedPostSaveable(),
 			isPublishable: forceIsDirty || isEditedPostPublishable(),
+			hasPublishAction: get( getCurrentPost(), [ '_links', 'wp:action-publish' ], false ),
```

Comment:
Should we introduce a new selector which would hide all those implementation details?
```js
hasPublishAction: hasCurrentPostAction( 'publish' )
```